barretenberg/cpp/src/barretenberg/sumcheck/sumcheck_round.hpp

```diff
@@ -518,15 +519,15 @@ template <typename Flavor> class SumcheckVerifierRound {
     // also copy paste in PG
```

Comment:
would you delete these 3 lines of stale comments oks, it's probably leftovers of me talking to myself ;)